queries.sql

```diff
@@ -1,3 +1,28 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * FROM animals WHERE name LIKE '%mon' ;
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-01' ;
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3 ;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu' ;
+SELECT name, escape attempts FROM animals WHERE weight_kg > 10.5 ;
+SELECT * FROM animals WHERE neutered = true ;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon') ;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
+BEGIN ;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon' ;
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+COMMIT
```

Comment:
- [ ] Please you did not begin this transaction so make sure that you begin this transaction, and verify that the change was made before and after you commit.